package.json

```diff
@@ -1,115 +1,137 @@
+        "storybook:copy-custom-styles": "node scripts/storybook/copy-custom-styles",
+        "storybook:build": "npm run storybook:copy-svg-icons && npm run storybook:copy-custom-styles && STORYBOOK_DS=4 build-storybook -o ./docs/ds4/storybook && STORYBOOK_DS=6 build-storybook -o ./docs/storybook",
+        "storybook:start-proper": "start-storybook -s .",
+        "storybook:start": "cross-env STORYBOOK_DS=6 npm-run-all -s storybook:copy-svg-icons storybook:copy-custom-styles storybook:start-proper"
+    },
+    "devDependencies": {
+        "@babel/cli": "^7",
+        "@babel/core": "^7",
+        "@babel/preset-env": "^7",
+        "@ebay/browserslist-config": "^1",
+        "@percy/storybook": "^3",
+        "@storybook/html": "^6",
+        "babel-loader": "^8",
+        "babel-plugin-transform-object-assign": "^6",
+        "browser-sync": "^2",
+        "btoa": "^1",
+        "core-js-pure": "^3",
+        "cross-env": "^7",
+        "details-element-polyfill": "^2",
+        "eslint": "^7",
+        "eslint-config-ebay": "^1",
+        "eslint-config-prettier": "^7.1.0",
+        "gulp": "^4",
+        "gulp-banner": "~0.1.3",
+        "gulp-flatten": "~0.4.0",
+        "gulp-less": "^4",
+        "gulp-rename": "^2",
+        "husky": "^4.3.8",
+        "jekyll": "^3.0.0-beta1",
+        "jsdom": "^16",
+        "lasso": "^3",
+        "lasso-autoprefixer": "^4",
+        "lasso-cli": "^2",
+        "lasso-less": "^3",
+        "less-loader": "^7",
+        "less-plugin-autoprefix": "^2",
+        "less-plugin-clean-css": "^1",
+        "lint-staged": "^10.5.3",
+        "makeup-active-descendant": "~0.3.9",
+        "makeup-expander": "~0.8.5",
+        "makeup-floating-label": "~0.1.2",
+        "makeup-key-emitter": "~0.1.3",
+        "makeup-modal": "~0.2.0",
+        "makeup-navigation-emitter": "~0.3.8",
+        "makeup-prevent-scroll-keys": "~0.0.4",
+        "makeup-roving-tabindex": "~0.3.7",
+        "makeup-switch-class": "~0.0.4",
+        "mkdirp": "^1",
+        "ncp": "^2",
+        "nodelist-foreach-polyfill": "^1",
+        "npm-run-all": "^4",
+        "percy": "~0.28.1",
+        "postcss-loader": "^4.0.3",
+        "prettier": "2.2.1",
+        "rimraf": "^3",
+        "stylelint": "^13",
+        "stylelint-config-prettier": "^8.0.2",
+        "stylelint-config-standard": "^20",
+        "stylelint-no-unsupported-browser-features": "^4",
+        "stylelint-order": "^4",
+        "stylelint-prettier": "^1.1.2",
+        "svgo": "^1",
+        "winston": "^3",
+        "yargs": "^16"
+    },
+    "keywords": [
+        "eBay",
+        "CSS",
+        "BEM",
+        "LassoJS",
+        "accessibility",
+        "WAI-ARIA",
+        "eBay MIND Patterns"
+    ],
+    "publishConfig": {
+        "registry": "https://registry.npmjs.org"
+    },
+    "browserslist": [
+        "extends @ebay/browserslist-config"
+    ],
+    "dependencies": {
+        "eslint-plugin-prettier": "^3.3.1"
```

Comment:
Should this be a devDependency rather than a dependency?

Reply:
yes, good catch